book/src/usage.md

```diff
@@ -42,7 +42,7 @@ helix. The keymappings have been inspired from [vim-sandwich](https://github.com
 `ms` acts on a selection, so select the text first and use `ms<char>`. `mr` and `md` work
 on the closest pairs found and selections are not required; use counts to act in outer pairs.
 
-It can also act on multiple seletions (yay!). For example, to change every occurance of `(use)` to `[use]`:
+It can also act on multiple selections (yay!). For example, to change every occurance of `(use)` to `[use]`:
```

Comment:
There's another one
```suggestion
It can also act on multiple selections (yay!). For example, to change every occurrence of `(use)` to `[use]`:
```

This is how I write without `set spell spelllang=en_us` :sweat_smile: